data/mods/blazemod/blaze_ammo.json

```diff
@@ -12,8 +12,11 @@
         "volume" : 1,
         "weight" : 3816,
         "bashing" : 8,
+        "cutting" : 0,
```

Comment:
All entries of type:
* "cutting" : 0
* "to_hit" : 0
* "pierce" : 0
* "range" : 0

Are redundant and should be removed.